src/controllers/csi/metadata/sqlite.go

```diff
@@ -49,7 +50,7 @@ const (
 
 	dynakubesAlterStatementMaxFailedMountAttempts = `
 	ALTER TABLE dynakubes
-	ADD COLUMN MaxFailedMountAttempts INT DEFAULT ` + defaultSqlMaxFailedMountAttempts + `;`
+	ADD COLUMN MaxFailedMountAttempts INT NOT NULL DEFAULT 0;`  // "Not null"-columns need a default value set
```

Comment:
spooky to have it at 0, but that should never happen

Reply:
should i set it to a different value?

Reply:
set it to 3